src/index.html

```diff
@@ -17,6 +17,49 @@
       </ul>
     </nav>
   </header>
+  <!-- POP - UP section -->
+  <div id="comments_popup" class="hidden">
+    <div id="popup_container" class="flex">
+      <div id="popup_picture_container">
+        <img src="#" alt="pokemon picture" id="popup_picture">
+        <h2 id="popup_title"></h2>
+      </div>
+      <div id="popup_close"><p>&#10060;</p></div>
+      <ul id="popup_details" class="flex">
+        <li class="detail_item flex">
+          <p class="detail_property">Height : </p>
+          <p id="property_height" class="detail_property">####</p><span>. in</span>
+        </li>
+        <li class="detail_item flex">
+          <p class="detail_property">Weight : </p>
+          <p id="property_weight" class="detail_property">####</p><span>. lb</span>
```

Comment:
I would recommend removing the dot from the span, it looks a little weird and it is good English practice to have heights and weights displayed in the following manner:
Height: 6 in  Weight: 75 lbs

Reply:
Hello @vechicin , Thanks for your comments and suggestion.

The requested changed have been implemented in the commit 8bce55809e87cb276229573bbfc839182d5217b5 which is inside the branch I'm actually working on.
https://github.com/MacCrazyman/pokemon-capstone/blob/8bce55809e87cb276229573bbfc839182d5217b5/src/style.css#L164-L165

https://github.com/MacCrazyman/pokemon-capstone/blob/8bce55809e87cb276229573bbfc839182d5217b5/src/index.html#L31

Best regards.

🧑‍🎄